File: app/src/debug/java/net/squanchy/support/debug/OnboardingResetter.java
package net.squanchy.support.debug;

import android.content.Context;
import android.content.SharedPreferences;

public class OnboardingResetter {

    private final SharedPreferences preferences;

    public OnboardingResetter(Context context) {
        preferences = context.getSharedPreferences("onboarding", Context.MODE_PRIVATE);
    }

    public void resetOnboarding() {
        preferences.edit()
                .clear()
                .apply();
    }
}

File: app/src/debug/java/net/squanchy/support/debug/OnboardingResetter.kt
package net.squanchy.support.debug

import android.content.Context
import android.content.SharedPreferences

internal class OnboardingResetter(context: Context) {

    companion object {

        private const val ONBOARDING_PREFERENCES_NAME = "onboarding"
    }

    private val preferences: SharedPreferences = context.getSharedPreferences(ONBOARDING_PREFERENCES_NAME, Context.MODE_PRIVATE)

    fun resetOnboarding() {
        preferences.edit()
                .clear()
                .apply()
    }
}
